src/parkapi_sources/converters/ellwangen/converter.py

```diff
@@ -0,0 +1,66 @@
+"""
+Copyright 2024 binary butterfly GmbH
+Use of this source code is governed by an MIT-style license that can be found in the LICENSE.txt.
+"""
+
+from datetime import datetime, timezone
+from typing import Any
+
+from openpyxl.cell import Cell
+
+from parkapi_sources.converters.base_converter.push import NormalizedXlsxConverter
+from parkapi_sources.models import SourceInfo
+
+
+class EllwangenPushConverter(NormalizedXlsxConverter):
+
+    source_info = SourceInfo(
+        uid='ellwangen',
+        name='Ellwangen Parkdaten',
+        public_url='https://www.ellwangen.de',
+        has_realtime_data=False,
+    )
+
+    header_row: dict[str, str] = {
```

Comment:
Why do you redefine the whole `header_row` when just one line has changed?

Reply:
If that's the new template this is fine.

Reply:
To simplify the converter, i have redefined the ```header_row``` as whole